src/OrchardCore/OrchardCore.DisplayManagement/Notify/Notifier.cs

```diff
@@ -17,26 +16,16 @@ public Notifier(ILogger<Notifier> logger)
             _logger = logger;
         }
 
-        public void Add(NotifyType type, LocalizedHtmlString message)
-        {
-            throw new System.NotImplementedException();
-        }
-
         public ValueTask AddAsync(NotifyType type, LocalizedHtmlString message)
         {
-            if (_logger.IsEnabled(LogLevel.Information))
-            {
-                _logger.LogInformation("Notification '{NotificationType}' with message '{NotificationMessage}'", type, message.ToString());
-            }
+            _logger.LogInformation("Notification '{type}' with message '{message}'", type, message);
```

Comment:
@sebastienros is there a down side from this change? I don't think there is because we are not building a string.

Reply:
It's fine. Use PascalCase for `{Type}` and `{Message}` though. (convention in dotnet).

Reply:
This is somewhat of a breaking change because log queries that fetched these messages and filtered on e.g `NotificationType` will break. It shouldn't be a big deal (if anybody relies on this, to begin with, but we can't really know), but there's not much point in changing this either.

Reply:
you mean the `NotificationType` to `Type` or removing the if statement?

Reply:
NotificationType to Type.

Reply:
okay. changed back